pkg/template_generator.go

```diff
@@ -314,7 +384,16 @@ func addContainerValue(key string, s1 string, s2 string) string {
 }
 
 func addTemplateImageValue(containerName string, image string, key string, containerValue map[string]interface{}) string {
-	img := strings.Split(image, ":")
+	var img []string
+	var repo string = ""
+	if strings.Contains(image, "/") {
```

Comment:
@TamalSaha I took your latest code, working fine for me as needed. Thanks.
Conflicts resolved.